marathon-bigip-ctlr.py

```diff
@@ -563,34 +716,31 @@ def create_config_marathon(bigip, apps):
         logger.debug(app.__hash__())
 
     logger.info("Generating config for BIG-IP")
-    f5 = {}
-    # partitions this script is responsible for:
-    partitions = frozenset(bigip.get_partitions())
+    services = {
```

Comment:
marathon-biigip-ctlr does not create Virtuals Addresses separate from Virtual Servers.